CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.27.0 (Unreleased)
```

Comment:
Let's start putting dates on the changelog entries.
Also, any reason to not release this?

Reply:
Not necessarily disagreeing on adding the changelog dates, but the canonical information for release-dates is available from crates.io via
```
curl 'https://crates.io/api/v1/crates/ironoxide' 
```

As for release - I think it's fine to release this whenever, but we should merge this PR first and then follow https://github.com/IronCoreLabs/ironoxide/blob/main/RELEASING.md 
If we want to start adding dates in CHANGELOG.md, adding it to the release checklist would be the right place.